app/code/Magento/InventorySales/Plugin/StockState/CheckQuoteItemQtyPlugin.php

```diff
@@ -163,4 +164,25 @@ private function getNumber($qty)
 
         return $qty;
     }
+
+    /**
+     * Returns websiteId assigned to store when scopeId is passed
+     * Returns default websiteId if no scopeId is passed
+     * @param int $scopeId
+     * @return string
+     */
+    private function resolveWebsiteId($scopeId = null) {
+        if($scopeId == null) {
```

Comment:
It's best to check with `===` to avoid weird side-effects in the future

Reply:
Fixed